resources/content_type.ts

```diff
@@ -25,7 +27,9 @@ const data = {
   'QuestBattles': 7,
   'Raids': 5,
   'RetainerVentures': 18,
+  'SaveTheQueen': 29,
   'TheHunt': 33,
+  'TheMaskedCarnivale': 27,
```

Comment:
We had a synthetic type for this in `zone_overrides.ts` that should get removed now that it's here, as it's causing a duplicate.